spec/requests/regions_spec.rb

```diff
@@ -113,40 +113,10 @@
   end
 
   context "delete", :delete do
-    it "can delete a region with POST" do
-      api_basic_authorize action_identifier(:regions, :delete)
-      region = FactoryBot.create(:miq_region)
-
-      expect { post api_region_url(nil, region), :params => gen_request(:delete) }.to change(MiqRegion, :count).by(-1)
-      expect_single_action_result(:success => true, :message => /#{region.id}/)
-    end
-
-    it "can delete a region with DELETE" do
-      api_basic_authorize action_identifier(:regions, :delete)
-      region = FactoryBot.create(:miq_region)
-
-      expect { delete api_region_url(nil, region) }.to change(MiqRegion, :count).by(-1)
-      expect(response).to have_http_status(:no_content)
-    end
-
-    it "can delete multiple regions with POST" do
-      api_basic_authorize action_identifier(:regions, :delete)
-      regions = FactoryBot.create_list(:miq_region, 2)
-
-      options = [
-        {"href" => api_region_url(nil, regions.first)},
-        {"href" => api_region_url(nil, regions.last)}
-      ]
-
-      expect { post api_regions_url, :params => gen_request(:delete, options) }.to change(MiqRegion, :count).by(-2)
-      expect_multiple_action_result(regions.size)
-    end
-
-    it "forbids deletion of a region without an appropriate role" do
-      expect_forbidden_request do
-        region = FactoryBot.create(:miq_region, :description => "Current Region description")
-        delete(api_region_url(nil, region))
-      end
+    it "forbids deletion of a region" do
+      region = FactoryBot.create(:miq_region, :description => "Current Region description")
+      delete(api_region_url(nil, region))
+      expect(response).to have_http_status(:not_found)
```

Comment:
404 seems like the wrong response.  404 is meant for resources not being found, I think this should probably be 405 "Method not allowed"

Reply:
@bdunne, @abellotti - Is there a generic way to do this? I don't see a 405 raised anywhere else, though it makes sense.

Reply:
When an action is no longer exposed from the api.yml, the API will not create the route for it.  So the 404 is what's coming back from rails, not the api.

Reply:
Ok, not going to have that hold up this PR.